src/minecraft/MinecraftManager.js

```diff
@@ -29,8 +29,7 @@ class MinecraftManager extends CommunicationBridge {
       host: this.app.config.server.host,
       port: this.app.config.server.port,
       username: this.app.config.minecraft.username,
-      password: this.app.config.minecraft.password,
-      version: false,
+      version: "1.8.9",
```

Comment:
There is no benefit of setting the bot version to 1.8.9 - `version: false` will automatically detect the version the server is running on and attempt to match it. Later versions of minecraft support up to chat length 256 which is greater than 1.8.9's 100 character limit.